src/table.js

```diff
@@ -387,7 +386,6 @@ class Table extends TransactionRequest {
    */
   /**
    * @typedef {array} TableReadResponse
-   * @property {Table} 0 The new {@link Table}.
    * @property {array[]} 1 Rows are returned as an array of object arrays. Each
```

Comment:
As I understand this typedef describes a tuple (in this case with just one element), so shouldn't the index for the rows be 0?